tests/integration/framework/namespace_test.go

```diff
@@ -47,8 +48,14 @@ func TestNamespace(t *testing.T) {
 			}
 		})
 
-	// Check after run to see that the namespace is gone.
-	if err := env.WaitForNamespaceDeletion(namespaceName); err != nil {
-		t.Fatalf("WaitiForNamespaceDeletion failed: %v", err)
+	settings, err := core.SettingsFromCommandLine("framework_test")
```

Comment:
You can get the settings from the suite/test context as well.

Reply:
This is outside of the scope where we have the `ctx` though, so I don't think that would work? Since this is testing the namespace gets cleaned up after the test completes?

I suppose we could capture the ctx and store it during the test but that doesn't seem good

Reply:
This is worse. :) You're depending on an implementation detail of the test framework.

You can capture the settings only (not ctx) within the test and then use it on the outside. The settings should be struct-copied, so there is no memory sharing.

Reply:
Ok ill make a follow up PR since this got merged